.github/workflows/documentation-build.yml

```diff
@@ -0,0 +1,32 @@
+name: documentation-build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  documentation_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup TARDIS Environment 
+        uses: conda-incubator/setup-miniconda@v1 
+        with:
+            environment-file: tardis_env3.yml
+            activate-environment: tardis
+
+      - name: Install TARDIS
+        shell: bash -l {0}
+        run: python setup.py install 
+
+      - name: Build Sphinx Documentation
+        shell: bash -l {0}
+        run: python setup.py build_docs
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: docs/_build/html/
```

Comment:
Use `publish_branch: gh-pages` explicitly as described [here](https://github.com/marketplace/actions/github-pages-action#%EF%B8%8F-set-another-github-pages-branch-publish_branch). It's the default behavior, but it's clearer for people looking for the first time this pipeline.